src/main/java/com/artistsolutions/betting/controllers/TeamController.java

```diff
@@ -1,29 +1,45 @@
 package com.artistsolutions.betting.controllers;
-
-import com.artistsolutions.betting.dto.Team;
+import com.artistsolutions.betting.dto.TeamDTO;
+import com.artistsolutions.betting.entity.Team;
+import com.artistsolutions.betting.repository.TeamRepository;
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.web.bind.annotation.*;
+import java.util.List;
+import java.util.stream.Collectors;
 
 @RestController
 @RequestMapping("/teams")
 public class TeamController {
 
+    @Autowired private TeamRepository teamRepository;
+    @Autowired private ModelMapper modelMapper;
```

Comment:
This is not the best way to use the Dependency Injection in spring, but for now, it will do. I will create you a task to refactor this later, now just keep in mind that this ca be done better.